contracts/actions/aaveV3/AaveV3Supply.sol

```diff
@@ -134,25 +122,27 @@ contract AaveV3Supply is ActionBase, AaveV3Helper {
 
         lendingPool.supply(tokenAddr, _amount, _onBehalf, AAVE_REFERRAL_CODE);
 
-        if (_enableAsColl) {
-            lendingPool.setUserUseReserveAsCollateral(tokenAddr, true);
-        } else {
-            lendingPool.setUserUseReserveAsCollateral(tokenAddr, false);
-        }
+         (,,,,,,,,bool usageAsCollateral) = getDataProvider(_market).getUserReserveData(tokenAddr, _onBehalf);
```

Comment:
this function uses up almost 100k gas, but will not provide us any worthwhile info (imo just using this for logging it as an event is not very worth it)
For example the full supply action uses 150-200k gas